### PR TITLE
Bugfixing, polishing, some new features

### PR DESCRIPTION
**Pull request by Monkatraz**
This PR is very generic. It's mainly just things that have been bothering me but weren't bad enough to get their own issue and PR.

In no particular order, this PR includes:
- Fixed ftml's URLs for `user-info`
- Added a settings button to Sheaf, which I needed to test various buggy things
- Fixed Stylelint + Svelte, now formatting works in Svelte again
- Merged a bunch of old changesets and bumped some package versions
- Fixed some bugs with ftml linting (Sheaf)
- Added `wide` and `flipped` attributes to `<Toggle />` (needed for Sheaf settings)
- Fixed some theming issues with Sheaf
- Basically rewrote `./scripts/dev.js` - it now is much better behaved, along with having cleaner code.
- Merged and simplified all of the scattered Vite configuration into one file: `./scripts/vite-utils.js`
- Probably some other stuff I'm forgetting about

### CODE REVIEW

File: web/modules/ftml-wasm-worker/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+
+- Overdue versioning bump
+
+### Patch Changes
+
+- 5ee0dde63: Change how the FTML WASM is imported by default
+- Updated dependencies [5ee0dde63]
+- Updated dependencies
+- Updated dependencies [5ee0dde63]
+- Updated dependencies [5ee0dde63]
```
Comment (emmiegit) [inline web/modules/ftml-wasm-worker/CHANGELOG.md:15]: Why is this repeated multiple times?

File: web/scripts/dev.js [hunk 2]
```diff
@@ -20,90 +23,167 @@ const isSudo = args.includes("sudo")
...
-  const start = performance.now()
+// -- MAIN
+
+let shuttingDown = false
 
+async function startup() {
   const doBuild = await question("Build containers? This can take a while. [y/N] -> ")
-  linebreak()
 
   if (doBuild.trim().toLowerCase() === "y") {
```
Comment (emmiegit) [inline web/scripts/dev.js:33]:
```suggestion
  if (["y", "yes"].contains(doBuild.trim().toLowerCase()) {
```
maybe?

**Review by emmiegit — APPROVED** [review]

File: web/modules/ftml-wasm-worker/CHANGELOG.md (hunk 1 above)
Comment (Monkatraz) [reply web/modules/ftml-wasm-worker/CHANGELOG.md:15]: I have no idea lol. I guess Changesets got a little confused.

File: web/scripts/dev.js (hunk 2 above)
Comment (Monkatraz) [reply web/scripts/dev.js:33]: Would be `.includes()` but yeah. Will do in my next PR (I'm working on a TypeScript imports PR)

**Comment by Monkatraz** [comment]
thanks @ammongit
